### ISSUE TITLE
[Spell] Warlock - Shadow Ward

### ISSUE THREAD

**Issue by Asuschrist** [FeenixServerProject/Archangel_2.4.3_Bugtracker#3585]
Shadow ward doesn't get its absorbing effect increased by spell damage while it should.
Currently Shadow Ward absorbs the same ammount of the Shadow damage as the tooltip says. (875)

Quote from wowwiki:
Patch 2.3.0 (13-Nov-2007): Now gains additional benefit from spell damage bonuses. Base absorb value of rank 4 has been reduced (875 from 920).

Quote from gamepedia: 
Shadow Ward
Level warlock ability
12% of base mana
30 sec cooldown
Instant cast
Absorbs (X + ($SP * 0.807)) shadow damage. Lasts 30 sec.

So basically with 1000 spell damage it should absorb 875 + 1000*807 = 875+807 = 1682 Shadow damage.

**Comment by warlee** [comment]
Most likely the issue for all absorb effects, requie futher testing

**Comment by warlee** [comment]
Priest's absorb effect works properly, testing mages now

**Comment by devidao** [comment]
Fixed, coming soon on PTR.
